repos/system_upgrade/common/actors/scanfilestocopyintotargetuserspace/libraries/scanfilestocopyintotargetuserspace.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+from leapp.libraries.stdlib import api
+from leapp.models import CopyFile
+
+# Maps src location in the source system to the destination within the target system
+FILES_TO_COPY_IF_PRESENT = {
+    '/etc/hosts': '/etc/hosts'
+}
+
+
+def scan_files_to_copy():
+    """
+    Scans the source system and identifies files that should be copied into target userspace.
+
+    When an item to be copied is identified a message :class:`CopyFile` is produced.
+    """
+    files_to_copy = []
+    for src_path in FILES_TO_COPY_IF_PRESENT:
+        if os.path.isfile(src_path):
```

Comment:
@pirat89 I guess we are good to have just these files in the DB and not logging / reporting anything if it is not found, right?

Reply:
@Rezney Yep, in this case we are ok to have just the message.